scapy/arch/common.py

```diff
@@ -42,7 +42,7 @@ def _check_tcpdump():
         return False
 
     # On some systems, --version does not exist on tcpdump
-    return proc.returncode == 0 or output.startswith(b'Usage: tcpdump ')
+    return proc.returncode == 0 or output.startswith(b'Usage: tcpdump ') or output.startswith(b'tcpdump: unrecognized option')
```

Comment:
This line is longer than 79 characters. Please split it.

Reply:
I have fixed this, please review again, thanks :)